src/app/lib/button/button.component.ts

```diff
@@ -12,7 +12,9 @@ type ButtonColor = 'primary' | 'primary-alt1' | 'primary-alt2' | 'primary-alt3'
 
 @Component({
   selector: 'button[hc-button]',
-  template: '<ng-content></ng-content>',
+  template: `<i *ngIf="glyph" class="fa {{glyph}} button-glyph"></i>
+  <ng-content></ng-content>
+  <i *ngIf="dropdown" class="fa fa-chevron-down fa-fw button-dropdown"></i>`,
```

Comment:
I know there's been some discussion about not having font awesome as a dependency. I'm wondering what a solution would look like if we didn't have font awesome. I remember there being talk about shipping with our own font. That may be the way to go without having an explicit dependency. But this should work if we list FontAwesome as a PeerDependency.

Reply:
I really like this idea, I propose we wait a little for the library to mature before we do something like this. As we are probably going to have quite a bit of churn until we have a solid set of icons

Reply:
I totally agree @corykon - a few of us were talking about it this morning and that's definitely where we want to get to.  I think as a starting point for this library we'll include FontAwesome as a dependency - because honestly I don't think I could give you a comprehensive list of all the icons we'd need in a HCGlyphs font yet.  I've made a note to make that a discussion topic at our next Cashmere OpenSpace.